### PR TITLE
Token

### PR DESCRIPTION
**Pull request by tarasIV**

### CODE REVIEW

File: src/app/login/login/login.component.ts [hunk 1]
```diff
@@ -23,8 +23,20 @@ export class LoginComponent implements OnInit {
     this.authService.login(this.userName, this.password)
       .subscribe((result) => {
         this.errorMessage = null;
-        // TODO: redirect user to role-specific route
-        this.router.navigate(['/shell/admin-panel/']);
+        let route: string;
```
Comment (dmitryy90) [inline src/app/login/login/login.component.ts]: move repeated code to the function in auth service and call in needed places

File: src/app/login/login/login.guard.ts [hunk 2]
```diff
@@ -8,10 +8,22 @@ export class LoginGuard implements CanLoad {
   constructor(private _authService: AuthenticationService,
     private router: Router) { }
 
-  canLoad(route: Route): boolean {
+  canLoad(): boolean {
     if (this._authService.loggedIn()) {
-      // TODO: get role from user service
-      this.router.navigate(['/shell/admin-panel/']);
+      let route: string;
```
Comment (dmitryy90) [inline src/app/login/login/login.guard.ts]: see previous comment

File: src/app/login/login/login.component.ts (hunk 1 above)
Comment (tarasIV) [reply src/app/login/login/login.component.ts]: Function defaultRoute was created and code was moved to the fucntion

File: src/app/login/login/login.guard.ts (hunk 2 above)
Comment (tarasIV) [reply src/app/login/login/login.guard.ts]: Fixed

**Pull request merged**